interface/main/calendar/add_edit_event.php

```diff
@@ -211,8 +211,8 @@ function DOBandEncounter($pc_eid)
 {
      global $event_date,$info_msg;
      // Save new DOB if it's there.
-     $patient_dob = trim($_POST['form_dob'] ?? '');
-     $tmph = $_POST['form_hour'] + 0;
+     $patient_dob = trim( string: $_POST['form_dob'] ?? '');
```

Comment:
thanks for the fix @Dhruv110904, no need for the `string` here and it's throwing a styling error with the space in front of it anyways so can just be
```
$patient_dob = trim($_POST['form_dob'] ?? '');
```

Reply:
fixed this issue, check PR #7628